pkg/skaffold/build/sequence.go

```diff
@@ -57,3 +67,46 @@ func InSequence(ctx context.Context, out io.Writer, tags tag.ImageTags, artifact
 
 	return builds, nil
 }
+
+func buildLogWriter(out io.Writer, artifact *latest.Artifact) (io.Writer, func(err error), error) {
```

Comment:
Do you think It'd be worth it to move these functions into a more general package, since we'll be wanting to use similar logging solution in deploy and status check for #4517?

Reply:
started here: https://github.com/GoogleContainerTools/skaffold/pull/4563